content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md

```diff
@@ -15,6 +15,10 @@ versions:
   ghae: '*'
   ghec: '*'
 ---
+## What is a gists?
+
+A Gist is a simple way to share code snippets with others. Like a repository, each Gist has version control.
+
 ## About gists
 
 Every gist is a Git repository, which means that it can be forked and cloned. {% ifversion not ghae %}If you are signed in to {% data variables.product.product_name %} when{% else %}When{% endif %} you create a gist, the gist will be associated with your account and you will see it in your list of gists when you navigate to your {% data variables.gists.gist_homepage %}.
```

Comment:
I think it's useful to start with a simple statement of what a gist is. 👍 

But it feels like this belongs in the "About gists" section. We can't say "Like a repository" because each gist _is_ a repository, so let's remove that sentence:

```suggestion
## About gists

A gist is a simple way to share code snippets with others. Every gist is a Git repository, which means that it can be forked and cloned. {% ifversion not ghae %}If you are signed in to {% data variables.product.product_name %} when{% else %}When{% endif %} you create a gist, the gist will be associated with your account and you will see it in your list of gists when you navigate to your {% data variables.gists.gist_homepage %}.
```